driver.sh

```diff
@@ -306,6 +314,50 @@ mako_reactor() {
        "${@}"
 }
 
+build_via_tuxbuild() {
+  echo "tuxbuild"
+  # https://gitlab.com/Linaro/tuxbuild/-/issues/55
+  if [[ ${ARCH} == "x86_64" ]]; then
+    tux_arch=x86
+  else
+    tux_arch=${ARCH}
+  fi
+  # https://gitlab.com/Linaro/tuxbuild/-/issues/53
+  if [[ ${ARCH} == "arm64" ]]; then
+    tux_image=Image
+  else
+    tux_image=${image_name}
+  fi
+  # Example command:
+  # $ tuxbuild build \
+  #   --git-repo https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git \
+  #   --git-ref master --target-arch x86 --kconfig defconfig
+  #   --toolchain clang-9
+  # Example artifacts to play with:
+  # https://builds.tuxbuild.com/k4gifttltEVagDHstq0NoA/ x86 811
+  # https://builds.tuxbuild.com/neNnj76CSx8sDEbwlUHGbA/ arm64 31:06
+  # https://builds.tuxbuild.com/9ieKegqDeZBTCMaG3et7ig/ arm64
+  # https://builds.tuxbuild.com/rPqYnwW4B8xhi8i6atmcpg/ arm
+  cmd="tuxbuild build --git-repo ${url} --git-ref ${branch:=master} --target-arch ${tux_arch} --kconfig ${config} --toolchain clang-${LLVM_VERSION}"
+  echo "${cmd}"
+
+  # https://gitlab.com/Linaro/tuxbuild/-/issues/56
+  start=$(date +%s)
+  build_url=$(timeout 50m ${cmd} | grep -o https://builds.tuxbuild.com/.\* | head -n1)
```

Comment:
I am fairly certain Travis will timeout here because there will not be any output for ten minutes. I am not sure how to get around that, we'll have to brain storm (or maybe reach out to Travis support and see if there is any way for that restriction to be waved).

Reply:
Last time I contacted Travis CI support, they told me there is no way to change the 10-minute silence timeout. The easiest way to get around that restriction is to just print something every 10 minutes.